charts/consul/values.yaml

```
@@ -430,6 +430,13 @@ global:
     # `<helm-release-name>-consul-federation`.
     createFederationSecret: false
 
+    # The name of the primary datacenter.
+    primaryDatacenter: ""
+
+    # A list of addresses of the primary mesh gateways in the form <ip>:<port>.
```

Might be extra helpful to see an example.
```suggestion
    # A list of addresses of the primary mesh gateways in the form <ip>:<port>.
    # (e.g. ["1.1.1.1:443", "2.3.4.5:443"] 
```

I think this could be helpful because I'm not sure what the YAML parsing behavior would be if the user entered `[1.1.1.1:443]`. I'm not sure if it might think it's parsing a float, get confused, and throw a hard to parse error.